plugins/PostProcessingPlugin/scripts/DisplayFilenameAndLayerOnLCD.py

```diff
@@ -36,10 +36,10 @@ def execute(self, data):
             name = self.getSettingValueByKey("name")
         else:
             name = Application.getInstance().getPrintInformation().jobName       
-        lcd_text = "M117 " + name + " layer: "
+        lcd_text = "M117 " + name + " layer "
         i = 0
         for layer in data:
-            display_text = lcd_text + str(i)
+            display_text = lcd_text + str(i) + "!"
```

Comment:
Why are you shouting? (vestigial debug statement?)

Reply:
Ignore this commit - I have since opened a fresh commit with enhancements.